official/keras_application_models/benchmark_main.py

```diff
@@ -46,6 +46,10 @@
     # "nasnetmobile": tf.keras.applications.NASNetMobile,
 }
 
+# Number of synthetic images as the same number of images in ImageNet
+_IMAGENET_TRAIN_IMAGES = 1281167
+_IMAGENET_EVAL_IMAGES = 50000
```

Comment:
I wouldn't bother making this complicated for synthetic data. We can re-evaluate if we add real data processing, but for now, maybe a single flag that defaults to 1000?